src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -100,35 +65,68 @@ private static string CleanUpDocString(this string text)
             return sb.ToString();
         }
 
-        private static string GetMemberElement(FunctionDescriptor function,
-            string suffix, XDocument xml)
+        private static string GetMemberElement(
+            FunctionDescriptor function,
+            XmlReader xml,
+            DocumentElementType property,
+            string paramName = "")
         {
-            // Construct the entire function descriptor name, including CLR style names
-            string clrMemberName = GetMemberElementName(function);
+            var assemblyName = function.Assembly;
+
+            // Case for operators.
+            if (string.IsNullOrEmpty(assemblyName))
+                return String.Empty;
 
-            // match clr member name
-            var match = xml.XPathEvaluate(
-                String.Format("string(/doc/members/member[@name='{0}']/{1})", clrMemberName, suffix));
+            string assemblyNameWithoutType;
+            // E.g. ProtoGeometry.dll => ProtoGeometry
+            if (assemblyName.LastIndexOf(".") != -1)
+                assemblyNameWithoutType = assemblyName.Substring(0, assemblyName.LastIndexOf("."));
+            else
+                assemblyNameWithoutType = assemblyName;
```

Comment:
Why don't we keep `ProtoGeometry.dll` as it is, and not turn it into `ProtoGeometry`? Will that cause any issue?